pkgs/os-specific/linux/busybox/default.nix

```diff
@@ -77,7 +87,7 @@ stdenv.mkDerivation rec {
   '';
 
   postConfigure = lib.optionalString useMusl ''
-    makeFlagsArray+=("CC=${stdenv.cc.targetPrefix}gcc -isystem ${musl}/include -B${musl}/lib -L${musl}/lib")
+    makeFlagsArray+=("CC=${stdenv.cc.targetPrefix}gcc -isystem ${musl.dev}/include -B${musl}/lib -L${musl}/lib")
```

Comment:
Good to no force `gcc`.

Reply:
Good catch, thank you.  Forgive the response spam, or if my responses become short, but I'm going to respond to things as I deal with them so I can keep track of which comments I've addressed and which I haven't.  Bear with me please :).